### PR TITLE
Add logs to debug replies inconsistency issue

### PR DESCRIPTION
**Pull request by yuliasherman**

### CODE REVIEW

File: bftengine/src/bftengine/ClientsManager.cpp [hunk 1]
```diff
@@ -101,22 +103,31 @@ void ClientsManager::loadInfoFromReservedPages() {
     ConcordAssert(replyHeader->replyLength >= 0);
     ConcordAssert(replyHeader->replyLength + sizeof(ClientReplyMsgHeader) <= maxReplySize_);
 
-    auto& repliesInfo = indexToClientInfo_.at(e.second).repliesInfo;
     // YS TBD: Multiple replies for client batching should be sorted by incoming time
+    auto& repliesInfo = indexToClientInfo_.at(e.second).repliesInfo;
+    const auto clientId = e.first;
+    LOG_DEBUG(CL_MNGR, "Add/update reply message" << KVLOG(clientId, replyHeader->reqSeqNum));
```
Comment (toly-kournik) [inline bftengine/src/bftengine/ClientsManager.cpp:109]: The message can be more specific it the return value has taken in consideration. std::pair<iterator, bool>

Comment (yuliasherman) [reply bftengine/src/bftengine/ClientsManager.cpp:109]: What exactly do you like to add?

Comment (toly-kournik) [reply bftengine/src/bftengine/ClientsManager.cpp:109]: Was it added or updated

Comment (toly-kournik) [reply bftengine/src/bftengine/ClientsManager.cpp:109]: Also, although it's not related to you change, maybe change 
`for (std::pair<NodeIdType, uint16_t> e : clientIdToIndex_)` to  something like `for (auto const& [clientId, clientIdx]: clientIdToIndex_)` for readability ?

Comment (yuliasherman) [reply bftengine/src/bftengine/ClientsManager.cpp]: Got it

**Review by toly-kournik — APPROVED** [review]
LGTM

**Pull request merged**